stix_shifter_modules/cybereason/stix_transmission/error_mapper.py

```diff
@@ -0,0 +1,36 @@
+from stix_shifter_utils.utils.error_mapper_base import ErrorMapperBase
+from stix_shifter_utils.utils.error_response import ErrorCode
+from stix_shifter_utils.utils import logger
+
+error_mapping = {
+    "JSONDecodeError": ErrorCode.TRANSMISSION_QUERY_PARSING_ERROR,
+    "InvalidRequestException": ErrorCode.TRANSMISSION_QUERY_LOGICAL_ERROR,
+    "InternalServerErrorException": ErrorCode.TRANSMISSION_INVALID_PARAMETER,
+    "AuthenticationError": ErrorCode.TRANSMISSION_AUTH_CREDENTIALS
```

Comment:
there should be a http connection error handling for wrong hostname. I see the error code is mapped to `unknown` if I make call with wrong hostname

Reply:
Error code has been mapped as "ConnectionError" in error_mapper file.